backpack.tf extended sorting.user.js

```diff
@@ -871,6 +874,65 @@ class</a></li>
 
 		}
 
+		var cnSort = 0;
+
+		function sortByCraftNumber() {
+			const sortCN = (a, b) => {
+				try {
+					return Number($(a)[0].outerText.split("#")[1]) - Number($(b)[0].outerText.split("#")[1]);
+				} catch (e) {
+					return parseInt(a[0].split(" ")[1]) - parseInt(b[0].split(" ")[1]);
+				}
+			}
+			let sorts = [
+				(a, b) => sortCN(a,b),
+				(a, b) => sortCN(b,a)
+			]
```

Comment:
Your functionality switches ascending and descending every time you select the `Group by craft number`.
What is the idea behind it?
By default currently it switches the order of the groups when you select exactly the same `Group by ...`.
So should you click this sorting twice in a row, you get craftnumbers at the bottom with the numbers descending. 
If you run this sorting for the first time, then change to for example paints and then change back it also displays differently.
Is there a reason to have switching ordering?

Reply:
nope lol